httpclient-core/src/main/java/esa/httpclient/core/Listener.java

```diff
@@ -143,19 +143,17 @@ default void onAcquireConnectionFailed(HttpRequest request,
      *
      * @param request request
      * @param ctx     ctx
-     * @param readTimeout readTimeout
      */
-    default void onWriteAttempt(HttpRequest request, Context ctx, long readTimeout) {
+    default void onWriteAttempt(HttpRequest request, Context ctx) {
```

Comment:
`onAttemptWrite` or `beforeWrite`?

Reply:
Thanks for your suggestion! But i think `onAttemptWrite` is better than `beforeWriter`. 
In my opinion, the former can show the specific time point of trying write operation, but
the latter perfer to show all the time before writting which is a time horizon.